apps/server/src/modules/files-storage/interface/file.ts

```diff
@@ -0,0 +1,8 @@
+import { Readable } from 'stream';
+
+export interface IFile {
+	name: string;
+	buffer: Buffer | Blob | ReadableStream | Readable;
```

Comment:
It is possible to pass all the types, but do we want this?

Reply:
The idea was to make IFile generic, but yes we can remove unused types